shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -223,17 +224,28 @@ - (id)accessibilityContainer {
 // private methods
 
 - (CGSize)contentSizeInternal {
-  CGRect result;
   const SkRect& rect = _semanticsObject.node.rect;
+  float x, y, width, height;
+  float scrollExtentMax = _semanticsObject.node.scrollExtentMax == INFINITY
```

Comment:
You shouldn't check equality against INFINITY, there is the isfinite function https://en.cppreference.com/w/c/numeric/math/isfinite